libvips/foreign/nsgifload.c

```diff
@@ -116,6 +116,10 @@ typedef struct _VipsForeignLoadNsgif {
 	 */
 	int frame_count_displayable;
```

Comment:
This is just equal to `frame_count` now, let's remove this.

How about putting `nsgif_info_t *info;` here instead and setting it just after `nsgif_data_scan()`? You could use it in a lot of places.

Reply:
Done!